include/pgettext.php

```diff
@@ -162,25 +162,31 @@ function string_plural_select_default($n) {
 }}
 
 
+
 /**
- * Return installed languages as associative array
- * [
- * 		lang => lang,
- * 		...
- * ]
+ * @brief Return installed languages codes as associative array
+ *
+ * Scans the view/lang directory for the existence of "strings.php" files, and
+ * returns an alphabetical list of their folder names (@-char language codes).
+ * Adds the english language if it's missing from the list.
+ *
+ * Ex: array('de' => 'de', 'en' => 'en', 'fr' => 'fr', ...)
+ *
+ * @return array
  */
-function get_avaiable_languages() {
-	$lang_choices = array();
-	$langs = glob('view/lang/*/strings.php'); /**/
-
-	if(is_array($langs) && count($langs)) {
-		if(! in_array('view/lang/en/strings.php',$langs))
-			$langs[] = 'view/lang/en/';
-		asort($langs);
-		foreach($langs as $l) {
-			$t = explode("/",$l);
-			$lang_choices[$t[2]] = $t[2];
+function get_available_languages() {
+	$langs = array();
+	$strings_file_paths = glob('view/lang/*/strings.php');
```

Comment:
I found out that nano syntax highlighting is pretty simple. The syntax highlighting doesn't do any string tokenization, it's just simple coloring regular expressions on top of each other. Which means that nano will color according to the multiline comment regular expression even if a portion of text has already been processed by the strings regular expression.

You can somewhat fix this problem in your nano syntax highlighting config file by moving the string regular expression lower than the multiline comment regular expression. This way the comments won't be highlighted in the strings, but the string will be highlighted in the multi-line comments.

I spent some time figuring if other editors had the same issue, and vim and emacs are correctly highlighting the code above. As far as I'm concerned, it's an editor-specific issue that should not impact the code.